DEVELOPERS_GUIDE.md

```diff
@@ -0,0 +1,119 @@
+# Developers Guide
+The contents fo this document are meant to help a developer understand the system requirements, coding guidelines, and operation of the CMR needed to work on it as an active contributer.
+
+
+**Table of content:**
+ - [Technical Stack](#technical-stack)
+ - [Running CMR](#running-the-cmr)
+
+
+## Technical Stack
+The CMR utilizes the following technologes to work:
+- [Java](https://docs.oracle.com/en/java/)
+- [Clojure](https://clojure.org)
+- [Lieningen](https://leiningen.org)
+- [Docker](https://www.docker.com)
+- [Elastic](https://www.elastic.co)
+- [Oracle](https://www.oracle.com/database)
```

Comment:
it might not be bad to add redis to the list since the server is launched and if people have redis viewer tools it will help them debug. Also a pointer to Kibana as an elastic viewer is helpful.